content/applications/websites/ecommerce/getting_started/catalog.rst

```diff
@@ -1,37 +1,58 @@
-==================================
-How to customize my catalog page 
-==================================
+=================
+Customize catalog
```

Comment:
In my opinion, merging "How to build a product page" and this one would make sense.
Both documentations are about creating and personalizing a product's page. Plus, the "How to build a product page" documentation is clearly outdated. It has been the same documentation for at least three different versions.

Having one single, complete, and well-written documentation with the whole process in one single place would be great (and better than having two short documentations).

Reply:
I completely agree. I went ahead and combined this doc with my other updated doc about product pages (#896), which I have turned into a draft for the time being. If this looks okay to you, just let me know, and I'll delete #896. Thanks!